packages/cysync-core/src/dialogs/AddAccountGuide/Dialogs/index.ts

```diff
@@ -1,3 +1,3 @@
 export * from './Asset';
-export * from './Device';
+export * from './X1 Vault';
```

Comment:
Keep this `Device`, `X1 Vault` is only for display texts, not for folder and file names.

Reply:
Done